entries/p/pirateship.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Pirate Ship": {
+    "domain": "pirateship.com",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://support.pirateship.com/en/articles/9931109-what-is-two-factor-authentication-2fa-on-pirate-ship",
+    "categories": [
+      "postal"
+    ]
```

Since Pirate Ship only allows for shipments within or originating from the U.S., it would make sense to add a region restriction
```suggestion
    ],
    "regions": [
      "us"
    ]
```